server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1929,7 +1930,8 @@ public static String createEmptyTranslog(
             },
             BigArrays.NON_RECYCLING_INSTANCE,
             DiskIoBufferPool.INSTANCE,
-            (d, s, l) -> {}
+            TranslogConfig.NOOP_OPERATION_LISTENER,
+            true
```

Comment:
Empty translog is always fsynced. Not sure if it is important to make it configurable as well